### ISSUE TITLE
Fix METviewer image creation via Docker build

### ISSUE THREAD

**Issue by JohnHalleyGotway** [NCAR/container-dtc-nwp#96]
While working on development of GitHub actions for #48, I found that running docker build for METviewer no longer works. This task is to update the METviewer Dockerfile(s) to get it building again. Note that there are 3 Dockerfile variants in [this directory](https://github.com/NCAR/container-dtc-nwp/tree/main/components/metviewer/METviewer). Please fix all 3.

The command to build METviewer listed as option 2 on [this page](https://dtcenter.org/nwp-containers-online-tutorial/software-containers/metviewer-database-display) now fails. Please see this build log for more details.
```
cd ${PROJ_DIR}/container-dtc-nwp/components
cd metviewer/METviewer 
docker build -t dtcenter/nwp-container-metviewer:${PROJ_VERSION} .
```
And here's the error at the end:
```
The command '/bin/sh -c yum -y update  && yum -y install epel-release  && yum -y install 'dnf-command(config-manager)'  && yum-config-manager --enable PowerTools' returned a non-zero code: 1
```
See [docker_build_metviewer_20220809.txt](https://github.com/NCAR/container-dtc-nwp/files/9294399/docker_build_metviewer_20220809.txt)

**Comment by TatianaBurek** [comment]
I successfully built the Docket image using feature_93-aws_port branch

**Comment by fossell** [comment]
@JohnHalleyGotway - It looks like GH Actions built the new image successfully too from PR #95 (although there was still an error doing "docker exec -i metviewer /scripts/common/metv_load_all.ksh mv_sandy).  But does the success build satisfy this Issue?

**Comment by fossell** [comment]
Fixed with #93
